LRQA-34313 investigate why test is not in the guest page

pcwhite opened the pull request.

pyoo47 committed:
LRQA-34625 Remove jgit from build.gradle

pyoo47 committed:
LRQA-34625 Remove jgit imports

pyoo47 committed:
LRQA-34625 Replace jgit RemoteConfig with GitRemote class

pyoo47 committed:
LRQA-34625 Remote jgit classes from createLocalBranch()

pyoo47 committed:
LRQA-34625 Remote jgit classes from deleteLocalBranch()

pyoo47 committed:
LRQA-34625 Rework exception handling

pyoo47 committed:
LRQA-34625 Simplify bash command execution to reduce code duplication

pyoo47 committed:
LRQA-34625 Rework pushToRemote() and rebase()

liuxingye committed:
LRQA-34444 Add the file of governors workflow

pyoo47 committed:
LRQA-34625 Implement BashCommandResult class

pyoo47 committed:
LRQA-34625 Rework remoteRemote() and change name to remoteGitRemote

pyoo47 committed:
LRQA-34625 Rework reset()

pyoo47 committed:
LRQA-34625 Simplify GitRemote constructor

pyoo47 committed:
LRQA-34625 Moving getRemoteBranch() out of GitRemote up to GitWorkingDirectory

pyoo47 committed:
LRQA-34625 Rename for clarity

pyoo47 committed:
LRQA-34625 These logging statements are redundant if the bash commands are already being printed in the log

pyoo47 committed:
LRQA-34625 Rework rebase()

pyoo47 committed:
LRQA-34625 Rename BashCommandResult -> ExecutionResult

pyoo47 committed:
LRQA-34625 Use a Branch instead of String when identifying the target branch to be checked out

pyoo47 committed:
LRQA-34625 Simplify Remote

pyoo47 committed:
LRQA-34625 Simplify branch logic

pyoo47 committed:
LRQA-34625 Keep backward compatibility with old clean() signature

pyoo47 committed:
LRQA-34625 Remove unused variable

pyoo47 committed:
LRQA-34625 Support fetching the remote without specifying a branch

pyoo47 committed:
LRQA-34625 createLocalBranch() should return the new Branch

pyoo47 committed:
LRQA-34625 Change parameter from branchName to branch

pyoo47 committed:
LRQA-34625 Don't throw an exception if branch doesn't exist. Support deleting local branches by name

pyoo47 committed:
LRQA-34625 Remove "remote" and "local" from branch method names. Callers can get local information by passing null for the remote parameter.

pyoo47 committed:
LRQA-34625 Use Branches instead of branchNames

Commit:
LPS-74502 Handle all exceptions that occur before even initial publish is started

Commit:
LPS-74502 Staging settings are not administered by SiteAdminPortlet anymore

pyoo47 committed:
LRQA-34625 Remove "remote" and "local" from branch method names. Callers can get local information by passing null for the remote parameter.

pyoo47 committed:
LRQA-34625 Both versions of createLocalBranch() should return a Branch

pyoo47 committed:
LRQA-34625 Bug fixes

pyoo47 committed:
LRQA-34625 Integrate changes to GitWorkingDirectory

pyoo47 committed:
LRQA-34625 Prevent invalide marker exceptions

pyoo47 committed:
LRQA-34625 Restore missing methods

pyoo47 committed:
LRQA-34625 Integrate changes to GitWorkingDirectory

Preston-Crary committed:
LPS-74693 Apply.

pyoo47 committed:
LRQA-34625 Prevent infinite loops

adolfopa committed:
LPS-74806 Don't show mount points when selecting root folders

adolfopa committed:
LPS-74806 Disable comments in CMIS repository

adolfopa committed:
LPS-74806 Hide Permissions action for external entries

adolfopa committed:
LPS-74806 Simplify conditionals and reduce scope of variables

adolfopa committed:
LPS-74806 Remove duplication

adolfopa committed:
LPS-74806 Update dependencies

pyoo47 committed:
LRQA-34625 Implement BufferedProcess to support bash commands with extended output

pyoo47 committed:
LRQA-34625 Add standard error input to exception message

pyoo47 committed:
LRQA-34625 Set MAX_THREAD_POOL_SIZE to 10

pyoo47 committed:
LRQA-34625 Rename for consistency

pyoo47 committed:
LRQA-34625 Simplify

pyoo47 committed:
LRQA-34625 Catch TimeoutException

pyoo47 committed:
LRQA-34625 Remove empty entries and add debug statement to troubleshoot future problems.

pyoo47 committed:
LRQA-34625 Break out of loop if successful

pyoo47 committed:
LRQA-34625 Remote logic fix

pyoo47 committed:
LRQA-34625 Increase timeout on git ls-remote calls

pyoo47 committed:
LRQA-34625 Handle case where sometimes 'git remote -v' standard out is blank

pyoo47 committed:
LRQA-34625 Print standardOut for 'git rev-parse --abbrev-ref' calls

pyoo47 committed:
LRQA-34625 In case current checked out branch is invalid.

pyoo47 committed:
LRQA-34625 Sort

anthony-chu committed:
LRQA-35061 Update navigation to Control Panel portlets

anthony-chu committed:
LRQA-35061 Update navigation to Site Administration portlets

anthony-chu committed:
LRQA-35061 Update navigation to User portlets

pavel-savinov committed:
LPS-74879 Add site navigation admin portlet module

pavel-savinov committed:
LPS-74879 Site navigation admin portlet

adolfopa committed:
LPS-74806 Don't create assets for external repository entries

pavel-savinov committed:
LPS-74879 Navigation bar

pavel-savinov committed:
LPS-74879 Management bar

pavel-savinov committed:
LPS-74879 Search methods implementation

pavel-savinov committed:
LPS-74879 Regenerate service code

pavel-savinov committed:
LPS-74879 Permissions for navigation menu and menu items

pavel-savinov committed:
LPS-74879 Navigation menu comparators

ealonso committed:
LPS-74942 Avoid null pointer exceptions

pavel-savinov committed:
LPS-74879 Add search container

pavel-savinov committed:
LPS-74879 Search container row action

pavel-savinov committed:
LPS-74879 Add button to be able to add new menu

pyoo47 committed:
LRQA-34625 Rename

pavel-savinov committed:
LPS-74879 Missing fetch & update methods, resources and permission checks

pavel-savinov committed:
LPS-74879 Add navigation menus panel app

pavel-savinov committed:
LPS-74879 Ability to add new navigation menu

pavel-savinov committed:
LPS-74879 Ability to delete selected navigation menus

shuyangzhou committed:
LPS-74894 Remove usages of StringUtil.valueOf()

vicnate5 committed:
LRQA-35133 Partial assert of the preview title

Commit:
LPS-74323 Add test

Commit:
LPS-74323 Semantic Versioning

Commit:
LPS-74323 Override imported record version

ealonso committed:
LPS-74944 Asset Publisher - Copy getDefaultAssetPublisherId and isDefaultAssetPublisher from AssetUtil

tinatian committed:
LPS-74896 Generated from https://github.com/rotty3000/rt.equinox.bundles/pull/20

mbowerman committed:
LPS-74922 Process DLFileEntryTypes during VerifyResourcePermissions since they can be added via direct SQL during the upgrade process

ealonso committed:
LPS-74944 Portal Search - Copy checkViewURL from AssetUtil

ealonso committed:
LPS-74944 Portal Search - SF

Commit:
LRAMF-410 Move necessary classes out of internal package

Commit:
LRAMF-410 Switch how URL mapper is set on portlet class: extend + inject

Commit:
LRAMF-410 Mapper must be extended to be configured the OSGi way

mtambara committed:
LPS-74899 Convert to a file to remove leading slash on Windows. Caused by 9f00c707171becedac884a2664787d99ef4c2786

Commit:
LRAMF-410 Service/model classes can only be accessed through registry

Commit:
LRAMF-410 JSPs should now go in alloy_mvc/ dir

ealonso committed:
LPS-74944 Asset Publisher - Don't use deprecated method

Preston-Crary committed:
LPS-74958 Break dependency of process package from ClassPathUtil, ClassPathUtil should be left in portal-kernel when we move to the reset of process to petra.

Ithildir committed:
LPS-74933 Bump up versions of styled/unstyled

Commit:
LRQA-34313 investigate why test is not in the guest page

wesleygong committed:
LRAMF-410 Source formatting

ealonso committed:
LPS-74944 Asset Taglib - Copy filterVocabularies from AssetUtil

ealonso committed:
LPS-74944 Asset Taglib - Uses new util class

ealonso committed:
LPS-74944 Asset Service - Copy filterCategoryIds, filterVocabularies and isValidWord

ealonso committed:
LPS-74944 Wiki - Copy toWord from AssetUtil

ealonso committed:
LPS-74944 Asset Taglib - Copy addPortletBreadcrumbEntries from AssetUtil

ealonso committed:
LPS-74944 Deprecate old class

ealonso committed:
LPS-74944 Asset Taglib - Uses new methods in util class

ealonso committed:
LPS-74944 Creates Asset Helper interface only with useful methods from AssetUtil

ealonso committed:
LPS-74944 Copy implementation from AssetUtil, no logic changes

ealonso committed:
LPS-74944 Makes methods private

ealonso committed:
LPS-74944 Use @Reference

ealonso committed:
LPS-74944 SF

ealonso committed:
LPS-74944 Uses dynamic data mapping classes instead of kernel

ealonso committed:
LPS-74944 Asset Publisher - Uses new util class

ealonso committed:
LPS-74944 build.gradle

inacionery committed:
LPS-74949 Add WorkflowDefinitionTitleException

rafaprax committed:
LPS-74323 SF

rafaprax committed:
LPS-74323 Rename method

ealonso committed:
LPS-74879 Don't set the icon

ealonso committed:
LPS-74879 buildLang

ealonso committed:
LPS-74879 We don't need data-qa-id in this case

ealonso committed:
LPS-74879 Adds box like in other edit entries

ealonso committed:
LPS-74879 We don't need to expose this class

rafaprax committed:
LPS-74323 SF

ealonso committed:
LPS-74879 Uses filter methods

ealonso committed:
LPS-74879 SF

ealonso committed:
LPS-74879 Move code to principal jsp

ealonso committed:
LPS-74879 Reorder to make it consistent with other Impl classes

ealonso committed:
LPS-74879 SF

ealonso committed:
LPS-74879 SF

ealonso committed:
LPS-74879 Copy from fragments, no logic changes

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master).

Commit:
LRAMF-410 Semantic versioning

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2029/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2029/jenkins-report.html">here</a>.</b></p>

Ithildir committed:
LPS-74933 Merge package.json from the root dir when publishing

Ithildir committed:
LPS-74933 Semver gradle-plugins-node

Ithildir committed:
LPS-74933 Use latest gradle-plugins-node

Ithildir committed:
LPS-74933 Use latest gradle-plugins-gulp

Ithildir committed:
LPS-74933 Use latest gradle-plugins-js-module-config-generator

Ithildir committed:
LPS-74933 Use latest gradle-plugins-js-transpiler

Ithildir committed:
LPS-74933 Run "publishNodeModule" tasks even publishing snapshots

Ithildir committed:
LPS-74933 Extract method

Ithildir committed:
LPS-74933 Apply the same configuration for both OSGi modules and themes

Ithildir committed:
LPS-74933 Move util methods

Ithildir committed:
LPS-74933 Always disable "publishNodeModule" for private projects

Ithildir committed:
LPS-74933 Remove duplicate constant

liferay-continuous-integration commented:
<html><h1>All tests PASSED.</h1><p>Build Time: 1 hour 56 minutes 16 seconds 179 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d7741cd977e6cb632f5217e8aa24fa5212d4e9b2">d7741cd977e6cb632f5217e8aa24fa5212d4e9b2</a></p>3 out of 3 jobs PASSED<details><summary><strong>3 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/2029/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/103070/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat80-mysql56-jdk8/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/1225/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-6.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2029/jenkins-report.html">here</a>.</h5></html>

Ithildir committed:
LPS-74933 Set a prerelease version on NPM when publishing a snapshot

vicnate5 committed:
LRQA-34444 SF Definition file

Ithildir committed:
LPS-74867 Update changelog

Ithildir committed:
LPS-74912 SF

Ithildir committed:
LPS-74933 Partial revert of ec5d0c2776c55216e249b5dd61acbbdbcb0f1fdc

Ithildir committed:
LPS-74933 Update readme

shuyangzhou committed:
LPS-74897 Add petra-lang dependency first before applying usage on portal side, this is needed for jspc.

Ithildir committed:
LPS-74933 Update changelog

shuyangzhou committed:
LPS-74942 Be consistent

Commit:
LRAMF-410 Remove unused variables

Commit:
LRAMF-410 log should be final

adolfopa committed:
LPS-74806 Move service wrappers into relevant module

Subscriptions shouldn't have to know which components depend on
it. Move the wrappers into the relevant service module (with the
exception of WorkflowLink, as there's no module for it yet).

Commit:
LPS-74324 Cannot select Data Provider output for Forms autofill rule

laszlopap committed:
LPS-74990 Adding a WebProxy portlet to a site template cause site templates propagation errors

marcoscv committed:
LPS-74961 - Fix Inline forms with switches

adolfopa committed:
LPS-74806 Use field injection

Commit:
LPS-74806 Apply same logic for subscription

shuyangzhou committed:
LPS-0 Bump version

liferay-continuous-integration committed:
LPS-0 Clear version-override for modules/apps/forms-and-workflow/portal-workflow

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/portal-workflow/.gitrepo'.

brianchandotcom committed:
LPS-74933 prep next, use latest

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/web-proxy/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/site/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/staging/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/document-library/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/message-boards/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/subscription/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/static/portal-osgi-web/.gitrepo'.

brianchandotcom committed:
LPS-74879 wordsmith

brianchandotcom committed:
LPS-74879 regen

brianchandotcom committed:
LPS-74958 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/site-navigation/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/dynamic-data-lists/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/petra/.gitrepo'.

brianchandotcom committed:
LRAMF-410 rename

brianchandotcom committed:
LPS-74944 Typo

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/wiki/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-search/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/asset/.gitrepo'.

brianchandotcom committed:
LRQA-34625 use " instead of \'

brianchandotcom committed:
LRQA-34625 Like the word "Linux"

brianchandotcom committed:
LRQA-34625 Wordsmith

brianchandotcom committed:
LRQA-34625 Like the word "Linux"

brianchandotcom committed:
LRQA-34625 Wordsmith

brianchandotcom committed:
LRQA-34625 Typo?

brianchandotcom committed:
LRQA-34625 Like the word "Linux"

brianchandotcom committed:
LRQA-34625 Wordsmith

brianchandotcom committed:
LRQA-34625 Wordsmith

brianchandotcom committed:
LRQA-34625 prep next

brianchandotcom committed:
artifact:ignore sync-service 1.5.6 prep next

brianchandotcom committed:
artifact:ignore document-library-repository-cmis-impl 2.0.19 prep next

brianchandotcom committed:
artifact:ignore document-library-service 1.1.28 prep next

brianchandotcom committed:
artifact:ignore calendar-web 1.0.37 prep next

brianchandotcom committed:
artifact:ignore portal-rules-engine-sample-web 2.0.10 prep next

brianchandotcom committed:
portal-workflow-kaleo-runtime-scripting-impl packageinfo

brianchandotcom committed:
artifact:ignore alloy-mvc 2.0.0 prep next

brianchandotcom committed:
artifact:ignore configuration-admin-web 1.0.30 prep next

brianchandotcom committed:
artifact:ignore frontend-css-web 1.0.35 prep next

brianchandotcom committed:
artifact:ignore frontend-theme-favicon-servlet 1.0.0 prep next

brianchandotcom committed:
frontend-theme-favicon-servlet packageinfo

brianchandotcom committed:
artifact:ignore petra-model-adapter 1.0.2 prep next

brianchandotcom committed:
artifact:ignore portal-configuration-module-configuration 2.0.11 prep next

brianchandotcom committed:
artifact:ignore portal-scheduler 3.1.1 prep next

brianchandotcom committed:
artifact:ignore portal-store-jcr 2.0.9 prep next

brianchandotcom committed:
artifact:ignore portal-template-freemarker 2.0.29 prep next

brianchandotcom committed:
artifact:ignore portal-cluster-multiple 1.0.17 prep next

brianchandotcom committed:
artifact:ignore portal-dao-orm-custom-sql 1.0.9 prep next

brianchandotcom committed:
artifact:ignore users-admin-impl 1.0.13 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-jasper-plugins 1.0.3 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-servlet-context-helper 2.1.13 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-servlet-jsp-compiler 2.0.25 prep next

brianchandotcom committed:
artifact:ignore portal-osgi-web-wab-extender 2.0.21 prep next

brianchandotcom committed:
artifact:ignore export-import-web 1.0.31 prep next

brianchandotcom committed:
artifact:ignore layout-admin-web 1.0.49 prep next

brianchandotcom committed:
artifact:ignore product-navigation-taglib 2.0.11 prep next

brianchandotcom committed:
artifact:ignore site-item-selector-web 2.0.13 prep next

brianchandotcom committed:
artifact:ignore staging-lang 2.0.10 prep next

brianchandotcom committed:
Prep portla-kernel 2.51.0

brianchandotcom committed:
export-import-service packageinfo

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 9a59b2e023edf27cc2f0000786fc772f4bb8ecc8

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/collaboration/document-library/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/calendar/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/forms-and-workflow/portal-rules-engine/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/configuration-admin/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-css/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/frontend-theme/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/petra/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-configuration/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-scheduler/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-store/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/portal-template/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/foundation/users-admin/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/static/portal-osgi-web/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/sync/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/export-import/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/layout/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/product-navigation/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/site/.gitrepo'.

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/web-experience/staging/.gitrepo'.
